arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-dialog.tsx

```diff
@@ -165,6 +165,37 @@ export class IDEUpdaterDialog extends ReactDialog<UpdateInfo | undefined> {
     goToDownloadPageButton.focus();
   }
 
+  private appendDonateFooter() {
+    const footer = document.createElement('div');
+    footer.classList.add('ide-updater-dialog--footer');
+
+    footer.innerText = nls.localize(
+      'arduino/ide-updater/donateText',
+      'Open source is love, '
```

Comment:
It looks like the space between "Open source is love," and "donate to support us" is lost:

![image](https://github.com/user-attachments/assets/5fe6a493-5fd9-4f24-99c8-4cf3dce2bf72)